src/ui/components/ShowMoreCard/styles.scss

```diff
@@ -36,6 +36,11 @@
   }
 }
 
+.ShowMoreCard-expand-link {
```

Comment:
I am going to remove this actually. It does not look nice (I did not really pay enough attention in the screenshots, sorry!)